### PR TITLE
Migrate CICD pipeline

### PR DESCRIPTION
**Pull request by jlpadilla**
Issue: https://github.com/open-cluster-management/backlog/issues/389

### CODE REVIEW

File: COMPONENT_VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+1.0.0
```
Comment (zlayne) [inline COMPONENT_VERSION]: Does this need to match release version? (1.1.0)

File: Makefile [hunk 2]
```diff
@@ -1,160 +1,34 @@
 ###############################################################################
-# Licensed Materials - Property of IBM Copyright IBM Corporation 2017, 2019. All Rights Reserved.
+# Licensed Materials - Property of IBM Copyright IBM Corporation 2019. All Rights Reserved.
```
Comment (zlayne) [inline Makefile]: I assume these licenses are going away anyway?

File: build/Configfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
+VCS_URL ?=$(shell git config --get remote.origin.url)
+IMAGE_NAME =$(shell cat COMPONENT_NAME)
+IMAGE_DISPLAY_NAME =Console Header
+ARCH = $(shell uname -m)
+ifeq ($(ARCH), x86_64)
+	IMAGE_NAME_ARCH =$(IMAGE_NAME)-amd64
+else
+	IMAGE_NAME_ARCH =$(IMAGE_NAME)-$(ARCH)
+endif
+IMAGE_MAINTAINER =jpadilla@us.ibm.com
```
Comment (zlayne) [inline build/Configfile]: Need to use public email at the moment>?

File: build/Configfile [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+IMAGE_VENDOR =IBM
```
Comment (zlayne) [inline build/Configfile]: IBM or Red Hat?

File: COMPONENT_VERSION (hunk 1 above)
Comment (jlpadilla) [reply COMPONENT_VERSION]: No, component_version is independent of release version.

File: Makefile (hunk 2 above)
Comment (jlpadilla) [reply Makefile]: Yes, license/copyrights will change, but we are still waiting for the new direction. Not changing it yet.

File: build/Configfile (hunk 3 above)
Comment (jlpadilla) [reply build/Configfile]: I don't have a RedHat email yet. Will update later.

File: build/Configfile (hunk 4 above)
Comment (jlpadilla) [reply build/Configfile]: Changed to Red Hat.

**Comment by jlpadilla** [comment]
I'm merging the current progress so we can continue with the CICD pipeline integration.

Currently the docker image is created and it can be started.  We still need to continue working on deploying it to a cluster.

**Pull request merged**